core/src/main/java/io/questdb/cairo/wal/WalPurgeJob.java

```diff
@@ -408,7 +414,15 @@ protected boolean runSerially() {
         final long t = clock.getTicks();
         if (last + checkInterval < t) {
             last = t;
```

Comment:
Still updating `last` even when not run means that the job will be skipped until the next `checkInterval`. I think this is the right approach, but I do wonder if we should reduce `checkInterval`. At the moment it's set to run every 5 minutes by default (if I recall correctly).
If things are going to get it to skip runs maybe it should run more often by default?

Reply:
Fair enough.